pandas_datareader/binance/__init__.py

```diff
@@ -0,0 +1,102 @@
+import pandas as pd
+from pandas_datareader.base import _BaseReader
+import requests
+
+BINANCE_BASE_URL = "https://api.binance.com"
+
+
+class BinanceReader(_BaseReader):
+    """Get data for the given name from Binance."""
+
+    _format = None
+
+    _intervals = {
+        "ONEMINUTE": "1m",
+        "THREEMINUTE": "3m",
+        "FIVEMINUTE": "5m",
+        "FIFTEENMINUTE": "15m",
+        "THIRTYMINUTE": "30m",
+        "ONEHOUR": "1h",
+        "TWOHOUR": "2h",
+        "FOURHOUR": "4h",
+        "SIXHOUR": "6h",
+        "EIGHTHOUR": "8h",
+        "TWELVEHOUR": "12h",
+        "ONEDAY": "1d",
+        "THREEDAY": "3d",
+        "ONEWEEK": "1w",
+        "ONEMONTH": "1M",
+    }
+
+    def __init__(
+        self,
+        symbols=None,
+        start=None,
+        end=None,
+        retry_count=3,
+        pause=0.1,
+        session=None,
+        interval="ONEDAY",
+        limit=500,
+    ):
+        super(BinanceReader, self).__init__(
+            symbols=symbols,
+            start=start,
+            end=end,
+            retry_count=retry_count,
+            pause=pause,
+            session=session,
+        )
+        self._interval = interval
+        self._limit = limit
+
+    @property
+    def url(self):
+        """API URL"""
+        return BINANCE_BASE_URL + "/api/v1/klines"
+
+    def clean_data(self, data):
```

Comment:
Add leading underscore to this method as it's only used internally (``_clean_data``)

Reply:
I have successfully changed this to private and tested the code.